src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,55 @@
  */
 
 public class Plane {
+    private String model;
+    private String color;
+    private int speed;
+    private int weight;
+    private String brand;
 
+    private Plane(PlaneBuilder builder) {
+        model = builder.model;
+        color = builder.color;
+        speed = builder.speed;
+        weight = builder.weight;
+        brand = builder.brand;
+    }
+
+    public static class PlaneBuilder {
+        private String model;
+        private String color;
+        private int speed;
+        private int weight;
+        private String brand;
+
+        public PlaneBuilder setModel() {
```

Comment:
what are you setting here as well as in all other setters? you do not pass anything to methods